src/base/http/requestparser.cpp

```diff
@@ -123,14 +123,22 @@ bool RequestParser::parseStartingLine(const QString &line)
 
         // Parse GET parameters
 #ifndef QBT_USES_QT5
-        QListIterator<QPair<QString, QString> > i(url.queryItems());
+        QListIterator<QPair<QByteArray, QByteArray> > i(url.encodedQueryItems());
```

Comment:
Since we using c++11 now don't insert a space between two (three, four ...) brackets:
```c++
QListIterator<QPair<QByteArray, QByteArray>> i(url.encodedQueryItems());
```

And in other places too.